metadata-models/src/main/pegasus/com/linkedin/assertion/DatasetAssertionInfo.pdl

```diff
@@ -3,64 +3,81 @@ namespace com.linkedin.assertion
 import com.linkedin.common.Urn
 
 /**
-* Assertion attributes that are applicable to Dataset Assertions
+* Attributes that are applicable to single-Dataset Assertions
 **/
 record DatasetAssertionInfo {
+    /**
+    * The dataset targeted by this assertion.
+    */
+    @Relationship = {
+      "name": "Asserts",
+      "entityTypes": [ "dataset" ]
+    }
+    dataset: Urn
+
     /**
     * Scope of the Assertion. What part of the dataset does this assertion apply to?
-    * Declared optional to make it convenient to inline into other aspects like AssertionInfo without requiring
-    * additional nesting. Semantically required, if this is a Dataset Assertion.
     **/
-    scope: optional enum DatasetAssertionScope {
```

Comment:
is this backward compatible?

Reply:
No, but luckily we have no current integrations depending on any of these models. We will publish a notice with the release